package.json

```diff
@@ -99,7 +101,7 @@
     "serve": "npx serve --no-clipboard -l 3000 -- build/",
     "start": "vite dev",
     "test:e2e:headless": "cross-env E2E_HEADLESS=1 vitest --mode e2e",
```

Comment:
Should we just remove this task? What did we do for data.microbit.org? Let's align on whatever you feel makes sense.

Reply:
I guess there must also be some vite/vitest config that we don't need anymore.

Reply:
Makes sense. About removing a task, is this what you mean 8c5684f6cb524b038bba44d3bdb4527a700001de? Or did you mean removing something else?

Reply:
Yeah, either update it or remove it as makes sense to you. Should have said npm script not task. Resolving.